src/test/java/com/ericsson/gerrit/plugins/eiffel/handlers/DataBaseHandlerTest.java

```diff
@@ -0,0 +1,217 @@
+package com.ericsson.gerrit.plugins.eiffel.handlers;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.powermock.api.mockito.PowerMockito.mock;
+
+import java.io.File;
+import java.net.ConnectException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.UUID;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.mockito.BDDMockito;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import com.ericsson.gerrit.plugins.eiffel.exceptions.NoSuchElementException;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(fullyQualifiedNames = "com.ericsson.gerrit.plugins.eiffel.*")
+public class DataBaseHandlerTest {
+    private final String branch = "my_test_branch";
```

Comment:
Please make those variables static and in uppercase so it is easier to see in the code where they come from. Please also have a calid_branch and in incorrect_branch. This makes it easier to undertsnd that a faulty branch gives no response.

Reply:
done, i might have get you wrong though

Reply:
Much better as static.
If you feel for it you could ad a specific static for branch name if the test is supposed to fail, just to make it easier to read. But already better as static since you now see where the variable comes from.